### PR TITLE
faillock: add page

### PR DESCRIPTION
**Pull request by mmatous**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
1.5.2

### CODE REVIEW

File: pages/linux/faillock.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- List login failures of all users:
+
+`sudo faillock`
+
+- List login failures of the specified user:
+
+`sudo faillock --user {{user}}`
+
+- Reset specified user's failure records:
```
Comment (CleanMachine1) [inline pages/linux/faillock.md:14]:
```suggestion
- Reset the failure records of the specified user:
```

File: pages/linux/faillock.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# faillock
+
+> Display and modify authentication failure record files.
+> More information: <https://github.com/linux-pam/linux-pam>.
```
Comment (CleanMachine1) [inline pages/linux/faillock.md:4]:
```suggestion
> More information: <https://manned.org/faillock>.
```

The current link doesn't give details  about faillock. Can you find any better resources or we will have to default to a man page.

File: pages/linux/faillock.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- List login failures of all users:
+
+`sudo faillock`
+
+- List login failures of the specified user:
```
Comment (mfrw) [inline pages/linux/faillock.md:10]: How about:
```suggestion
- List login failures for the specified user:
```

**Comment by mmatous** [comment]
Accepted suggestions. There aren't better resources for faillock, as it's just a small part of pam. Manpages it is.

File: pages/linux/faillock.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# faillock
+
+> Display and modify authentication failure record files.
+> More information: <https://manned.org/faillock>.
+
+- List login failures of all users:
+
+`sudo faillock`
+
+- List login failures for the specified user:
```
Comment (CleanMachine1) [inline pages/linux/faillock.md:10]:
```suggestion
- List login failures of the specified user:
```

Personally I prefer of here @mfrw and i think we have used it more commonly in this context. What do you think, regardless they mean the same thing.

Comment (mmatous) [reply pages/linux/faillock.md:10]: Slight "of" preference. Accepting, considering it has thumbs up from @mfrw.

**Review by blueskyson — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
LGTM, thanks for your contribution!

**Pull request merged**